docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LightGbmAdvanced.cs

```diff
@@ -62,5 +62,13 @@ public static void Example()
             //   RMS: 7.17
             //   RSquared: 0.08
         }
+
+        public static void PrintMetrics(RegressionMetrics metrics)
+        {
+            Console.WriteLine($"Mean Absolute Error: {metrics.MeanAbsoluteError:F2}");
+            Console.WriteLine($"Mean Squared Error: {metrics.MeanSquaredError:F2}");
+            Console.WriteLine($"Root Mean Squared Error: {metrics.RootMeanSquaredError:F2}");
+            Console.WriteLine($"RSquared: {metrics.RSquared:F2}");
+        }
```

Comment:
for trainers, we should modify the .ttinclude template file, not the cs file directly.

Reply:
`*Advanced.cs` are not API samples. They are legacy end-to-end pipelines saved for future use.